test/run

```diff
@@ -0,0 +1,77 @@
+#! /usr/bin/make -f
```

Comment:
Please could you move this file to some subdirectory too?

Reply:
Done.